modules/core/test/v2/unit/baseCoin.ts

```diff
@@ -60,21 +76,26 @@ describe('V2 Base Coin:', function() {
   });
 
   describe('Token initialization', function() {
-    it('Tokens initialized with name and contract should be instances of Token', function() {
-      (basecoinTokenWithName instanceof Token).should.equal(true);
-      (basecoinTokenWithContractHash instanceof Token).should.equal(true);
+    it('ERC20 Tokens initialized with name and contract should be instances of Erc20Token', function() {
+      (basecoinErc20TokenWithName instanceof Erc20Token).should.equal(true);
```

Comment:
`basecoinErc20TokenWithName.should.be.instanceof(Erc20Token)`

Reply:
it reads better, changed